### PR TITLE
Ball Dodger Game 

### LINKED ISSUE #3423: [New game]: Falling ball game
### 🎮 Game Request

Game logic and basic description

- HTML, CSS, and Javascript are used in the creation of this project.
- One player may play this game by using the keyboard's arrow keys.
- The player's score will be kept track of until the player loses.
- If the ball touches the top bar, the player will lose the game.

### Point down the features

Game points

- When a player loses a game, the score will be shown 👍
- The player's greatest score will be stated.
- Putting graphic user interface (GUI) into practice to create a more interactive experience. 
Screenshot 2024-05-18 102233

- The game is still in development phase, will improve the UI of the game. 

### Select program in which you are contributing

GSSoC24

### Code of Conduct

- [X] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) of this project.

### PR DESCRIPTION
**Pull request by AshishPandey04**
## PR Description 📜

I have added a new game Ball Dodger (also known as falling ball game ) . I have also improve user experience by Allowing user to restart the game and also know their Score.

Fixes #3423

<hr>
 
## Mark the task you have completed ✅

- [✅ ] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [ ✅] I have performed a self-review of my own code or work.
- [ ✅] I have commented my code, particularly in hard-to-understand areas.
- [ ✅] My changes generates no new warnings.
- [ ✅] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [✅ ] I have added screenshot for website preview in assets/images 
- [ ✅] I have added entries for my game in main README.md
- [ ✅] I have added README.md in my folder 
- [ ✅] I have added working video of the game in README.md (optional)
- [ ✅] I have specified the respective issue number for which I have requested the new game.

<hr>

## Thank you soo much for contributing to our repository 💗

### CODE REVIEW

**Comment by kunjgit** [comment]
Thank you @AshishPandey04 ,for creating the PR and contributing to our GameZone 💗 
 Review team will review the PR and will reach out to you soon! 😇 
 Make sure that you have marked all the tasks that you are done with ✅. 
 Thank you for your patience! 😀

**Comment by AshishPandey04** [comment]
@kunjgit please check the PR

File: README.md [hunk 1]
```diff
@@ -107,8 +107,8 @@ This repository also provides one such platforms where contributers come over an
 
 </center>
 
-| Game                                                                                                             | Game                                                                                                    | Game                                                                                                         | Game                                                                                                               | Game                                                                                                        |
-| ---------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------ | ------------------------------------------------------------------------------------------------------------------ | ----------------------------------------------------------------------------------------------------------- | --- |
```
Comment (ishita-43) [inline README.md:111]: add these again and add your game below these at line number 112

**Review by ishita-43 — CHANGES_REQUESTED** [review]

**Comment by AshishPandey04** [comment]
I have resolved  conflicts , please have a look and let me know if there is any more changes required

File: README.md [hunk 2]
```diff
@@ -107,9 +107,9 @@ This repository also provides one such platforms where contributers come over an
...
+ |[Block Dodger](https://github.com/AshishPandey04/GameZone/tree/newBranch/Games/Block_Dodger)    | [Master Typing](https://github.com/kunjgit/GameZone/tree/main/Games/Master_Typing)                               | [Treasure Hunt](https://github.com/Antiquely3059/GameZone/tree/main/Games/Treasure%20Hunt)              | [Virtual Pet](https://github.com/Antiquely3059/GameZone/tree/main/Games/Virtual_Pet)                         | [MazeRunner](https://github.com/kunjgit/GameZone/tree/main/Games/MazeRunner)                                       | [Ping_Pong_Singleplayer](https://github.com/kunjgit/GameZone/tree/main/Games/Ping_Pong_Singleplayer)        |     |
```
Comment (ishita-43) [inline README.md:112]: change this with kunjgit instead ashish in the link

**Review by ishita-43 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (AshishPandey04) [reply README.md]: I have resolved it , Please have a look.
and let me know if I need to add anything.

**Review by ishita-43 — APPROVED** [review]
Approved!!

**Comment by kunjgit** [comment]
Thank you  @AshishPandey04 , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

**Comment by AshishPandey04** [comment]
I have resolved all the merge conflicts , now how to merge (i tried but it is showing up to date with main) .

**Comment by kunjgit** [comment]
Thank you  @AshishPandey04 , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

**Pull request merged**